### PR TITLE
Correct typo and line numbers

### PR DESCRIPTION
**Pull request by brotkrueml**

### CODE REVIEW

File: Documentation/ApiOverview/Typo3CoreEngine/UsingDataHandler/Index.rst [hunk 1]
```diff
@@ -129,7 +129,7 @@ Imagine the $data array something like this:
    );
 
 This aims to create two new pages in the page with uid "456". In the
-follow code this is submitted to the database. Notice how line 3
+following code this is submitted to the database. Notice how line 2
```
Comment (mbrodala) [inline Documentation/ApiOverview/Typo3CoreEngine/UsingDataHandler/Index.rst:132]: Here I'd directly mention `$dataHandler->reverseOrder = 1` instead, then we don't need to fix this over and over again. It also allows for understanding this without jumping to the code example.

File: Documentation/ApiOverview/Typo3CoreEngine/UsingDataHandler/Index.rst [hunk 2]
```diff
@@ -161,7 +161,7 @@ Both Data and Commands Executed With Alternative User Object
 
 In this case it is shown how you can use the same object instance to
 submit both data and execute commands if you like. The order will
-depend on the order of line 4 and 5.
+depend on the order of line 3 and 4.
```
Comment (mbrodala) [inline Documentation/ApiOverview/Typo3CoreEngine/UsingDataHandler/Index.rst:164]: Same here: I'd mention the two methods directly instead.

File: Documentation/ApiOverview/Typo3CoreEngine/UsingDataHandler/Index.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/ApiOverview/Typo3CoreEngine/UsingDataHandler/Index.rst]: I removed the line numbers altogether in this section, the code is self-explanatory.

File: Documentation/ApiOverview/Typo3CoreEngine/UsingDataHandler/Index.rst (hunk 2 above)
Comment (brotkrueml) [reply Documentation/ApiOverview/Typo3CoreEngine/UsingDataHandler/Index.rst]: same here

**Pull request merged**